interfaces/builtin/allegro_vcu.go

```diff
@@ -0,0 +1,63 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+// https://www.xilinx.com/support/documentation/ip_documentation/vcu/v1_2/pg252-vcu.pdf
+// https://github.com/Xilinx/vcu-modules/
+const allegroVcuSummary = `allows access to Xilinx Allegro Video Code Unit`
```

Comment:
mmm I'm not sold on the name "allegro-vcu", maybe something with xilinx in it since this seems to be a xilinx specific interface? Also it seems to be a "control" interface to me, since reading the documentation these device nodes don't just pipe video frames in/out of the system, the VCU can be controlled/configured through this as well.

- [ ] xilinx-vcu-control
- [ ] xilinx-allego-vcu-control
- [ ] xilinx-vcu-control
- [ ] xilinx-media-control
- [ ] xilinx-video-control

cc @pedronis for naming considerations

Reply:
The VCU itself comes from Allegro (https://www.allegrodvt.com/), Xilinx in this case just integrates. So we might eventually find these devices in other SoC. Re: -control postfix, not sure if I like that too much, maybe if you have a `vcu` and then a `vcu-control` interface that would make sense, otherwise it makes things more verbose than needed (what is control and what is not?)

Reply:
given the devices are called /dev/allegro*   it seems the name is ok,  I wonder if the summary should mention xinlinx so directly if the vcu can appear elsewhere anyway? and the devices themselves don't seem tied to xilink particularly?

Reply:
I wasn't aware that the VCU comes from Allego, in that case I think not mentioning xilinx in the name makes sense (bu still mention xilinix in the description as a reference device for us for this interface

Reply:
I've removed xilinx from the description now.